datajunction-server/datajunction_server/construction/build.py

```diff
@@ -530,6 +532,7 @@ def build_node(  # pylint: disable=too-many-arguments
         dimensions,
         orderby,
         limit,
+        include_dimensions_in_groupby=(node.type == NodeType.METRIC),
```

Comment:
I think this is good (for now), but I have a feeling that `get_query(...)` function becomes a little too overloaded. I think that it would be easier if we had a simple query generation method that behaves the same way for any node in DJ dag. And in addition to that we could have a special methods for generating SQL from multiple metrics and dimensions. And we have the latter one already. I am just not sure if we should make the `get_query` function do different things for different node types...

Reply:
Will merge this for now and we can decide on whether a different interface is needed afterwards